QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150428_001980	$According to the information I've gathered from the wings of Baibora, the demons come out in groups from a place called Demon's Prison. {nl}Please help us with this to prevent a greater chaos from happening.
+QUEST_20150428_001981	$Help the druids that were dispatched to the farm of the Green Family. {nl}They're looking for people who can help with the mass purification.
+QUEST_20150428_001982	$Dina Bee Farm is in danger of monster attacks. {nl}They can't even work so they desperately need help.
+QUEST_20150511_001983	$It's better to use the cable car than to head straight into the monsters.
+QUEST_20150511_001984	$Just don't believe everything he says. {nl}He was known for being quite the show-off back at home.
+QUEST_20150511_001985	$It would be nice if all these Revelators being here meant that the Goddesses are back.
+QUEST_20150511_001986	$Power is the most important attribute for swordsman. {nl}You could also train yourself to be agile and possess strong stamina. {nl}You should first think about what to become from now on.
+QUEST_20150511_001987	$Power is the most important attribute for archers. {nl}But, of course you can't ignore agile movement. {nl}You should first think about what to become from now on.
+QUEST_20150511_001988	$Intelligence is the most important attribute for Clerics. {nl}But sometimes you many need strong power or spirit. {nl}You should first think about what to become from now on. 
+QUEST_20150511_001989	$Helping reconstruct Fedimian may be more meaningful than pottery. {nl}But people need something to soothe their hearts.
+QUEST_20150511_001990	$If I had the diary, I could continue my adventure, but I wasn't selected… {nl}But if it were you…
+QUEST_20150511_001991	$If you cleverly use the dry air that forms around severe colds, you'd be able to freeze any object even in a desert. {nl}However, if you can't receive the blessing from the Goddess, it won't be possible to make that kind of air.
+QUEST_20150511_001992	$I'll report the damages. {nl}Chasing a Sparnas, a gigantic bird-type creature, will be hard. Despite being large, they are rather agile. {nl}It usually goes after the supplies, but I can't imagine nor find out why. I'm guessing someone's playing a trick.
+QUEST_20150511_001993	$Grain pocket was discovered in the ashes of the supplies. {nl}They smell so bad and old they can't be used for food at all. {nl}Please check the supplies for anything missing.
+QUEST_20150511_001994	$Defeat all the monsters nearby. {nl}The vicious energy in the vine forest is constantly creating new monsters. {nl}But you should first defeat most of them before you face Ironbaum. {nl}Even a small factor could ruin the big task. {nl}Also, you could try threatening the monsters so they can't do what they want to do.
+QUEST_20150511_001995	$(Everything is scribbled so poorly that they can't be distinguished from letters or drawings.)
```

Comment:
This kind of loses the original meaning for me.
"(everything is scribbled so poorly, one can't distinguish the letters from drawings.)"